src/browser/StorybookPage.ts

```diff
@@ -134,10 +134,19 @@ function emitSetCurrentStory(id: string) {
     );
   }
 
-  return new Promise((resolve) => {
-    // @ts-expect-error Access the protected "channel", so we can send stuff through it.
-    const channel = storybookPreview.channel;
+  // @ts-expect-error Access the protected "channel", so we can send stuff through it.
+  const channel = storybookPreview.channel;
 
+  // update global to disable known addons containing `axe-core`
+  channel.emit('updateGlobals', {
+    globals: {
+      a11y: {
+        manual: true,
+      },
+    },
+  });
```

Comment:
Nice find!

An alternative might be to add the `globals=a11y.manual:!true` query param to the `storybookUrl` when visiting the page: https://github.com/chanzuckerberg/axe-storybook-testing/blob/e6d919fcaa0a5bc2ac495980d8f62b6c370dfdcb/src/browser/TestBrowser.ts#L43

Reply:
Good call. I actually tried that first, but after the call to `channel.emit` it somehow lost the global (... or reset it?). Kinda weird, but since this relates to that little brittle section, opted to keep all the brittle bits close together for future debugging.